{"name": "test_login_fail", "status": "passed", "start": 1640492243090, "stop": 1640492243090, "uuid": "7206532e-2fb8-4078-8803-17700b85de78", "historyId": "e69c0f76fefae23c3f8503786dd6fa10", "testCaseId": "91343b1f193d6f659377003e69dc569d", "fullName": "test_four.TestLogin#test_login_fail", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "story", "value": "登录失败"}, {"name": "suite", "value": "test_four"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-N0R16UT"}, {"name": "thread", "value": "8248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_four"}]}